lib/key_processor.rb

```diff
@@ -6,6 +6,7 @@ def initialize(input_key)
     @key_shift_keys = "abcd"
     @counter = 0
     @key_shift_values = {}
+    process_key
```

Comment:
this was changed to have immediate access to key shift values upon instantiation. Maybe make a class method?